ChangeLog

```diff
@@ -1,54 +1,9 @@
-* Tue Jul 16 2024 Ken Gaillot <kgaillot@redhat.com> Pacemaker-2.1.8-rc4
-- 6 commits with 21 files changed, 126 insertions(+), 38 deletions(-)
-
-- Features added since Pacemaker-2.1.8-rc3
-  + tools: crm_verify now reports invalid fence topology levels
```

Comment:
I don't see this feature mentioned anywhere in the list of features since 2.1.7.

Reply:
oops. i also forgot a few things newer than rc4 ...

Reply:
updated commit